css/main.scss

```
@@ -103,5 +103,6 @@ $flagsImagePath: "../images/";
 @import 'e2ee';
 @import 'responsive';
 @import 'connection-status';
+@import "hd-video-alert";
```

Just for consistency:

```suggestion
@import 'hd-video-alert';
```